templates/nextjs/sway-programs/fuel-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "testnet"
+
+[components]
+forc = "0.60.0"
+fuel-core = "0.30.0"
```

Comment:
Out of curiosity, did you consider rendering this file on-demand based on `@fuels/versions`?

Reply:
I did but our version updates are still a fairly manual process so IMO the additional overhead here wasn't that big.